frontend/src/components/Chat/ChatContainer.tsx

```diff
@@ -20,21 +20,34 @@ const sampleMessage: Message = {
   directMessageId: undefined,
 };
 
-export default function ChatContainer(): JSX.Element {
-  return (
-    <div className='chat-container'>
-      <div className='scrollable-messages'>
-        {/* delete these when real messages replace them */}
-        <SingleMessage message={sampleMessage} myPlayerID='23124' />
-        <SingleMessage message={sampleMessage} myPlayerID='122342' />
-        <SingleMessage message={sampleMessage} myPlayerID='33333' />
-        <SingleMessage message={sampleMessage} myPlayerID='123234' />
-        <SingleMessage message={sampleMessage} myPlayerID='122325' />
-        <SingleMessage message={sampleMessage} myPlayerID='33333' />
-        <SingleMessage message={sampleMessage} myPlayerID='123235' />
-        <SingleMessage message={sampleMessage} myPlayerID='33333' />
-      </div>
-      <div className='chat-input'>input goes here</div>
+interface ChatContainerProps {
+  // so for now- Town/Proximity will pass in undefined
+  directMessageID: string | undefined;
+  chainType: MessageType;
+}
+
+export default function ChatContainer({
+  directMessageID,
+  chainType,
+}: ChatContainerProps): JSX.Element {
+  // optional passing of an direct message chain ID, depends on chain type
+  // if new DM - wait until first input to create the new DM chain -- using the id that was given as a prop
+  // eventually useCoveyAppState to connect to messages
+
+  const finalSentence = (chainType === MessageType.DirectMessage) ? `you want ${directMessageID}` : '';

```

Comment:
I'm not sure about this "you want [directMessageId" piece here. I think it might make more sense to display the username of the person you're chatting with?

Reply:
Yeah that kind of stuff was going to be updated during my next task/PR when I go through to display correct chats as well. I imagine that all of the text I added to ChatContainer would end up removed.

Reply:
I'm also an advocate for displaying the username since the ID is generated so that might be a bit confusing but other than that looks g!

Reply:
yep that's the plan for next step @MahemaS27